cmd/gcp-controller-manager/main.go

```diff
@@ -56,7 +56,6 @@ const (
 	leaderElectionResourceLockNamespace = "kube-system"
 	leaderElectionResourceLockName      = "gcp-controller-manager"
 
-	kubeconfigQPS     = 100
 	kubeconfigBurst   = 200
```

Comment:
Let's expose burst as well and pass it to `informerKubeconfig`.

Reply:
Done.